### ISSUE TITLE
`modules.mode` option is completely mysterious

### ISSUE THREAD

**Issue by gabriel-fallen** [webpack-contrib/css-loader#1234]
Readme says that possible values for the `modules.mode` option are `local`, `global`, and `pure`, but doesn't explain what behaviour this option controls and what's the difference between the possible values.

Documentation Is:

- [ ] Missing
- [x] Needed
- [ ] Confusing
- [ ] Not Sure?

**Comment by evilebottnawi** [comment]
https://github.com/webpack-contrib/css-loader#scope

**Comment by gabriel-fallen** [comment]
@evilebottnawi thanks, but I don't think that closes the issue.

First, `modules.mode` documentation still doesn't say it's related (and how) to scopes, and second, what's the meaning of `pure` option?

**Comment by evilebottnawi** [comment]
What exactly would you like to see there, describe, a lot of developers use it, and there were no problems, so I would like to understand in detail what you do not understand

**Comment by gabriel-fallen** [comment]
> What exactly would you like to see there

As I said before at least two things:

1. the fact that `modules.mode` is related to scopes &mdash; I didn't know that before you told me here. Ideally, exactly how it is related.

2. what happens if I set `modules.mode` to `pure`? I still has no idea.

> a lot of developers use it, and there were no problems

Yeah, sure and I'm one of those developers apparently as long as a project template I used have `css-loader` referenced and preconfigured. Indeed I had no problems &mdash; I even didn't know about `css-loader` &mdash; until I've updated dependencies and it stopped working. Now I have to read documentation and figure out what to do. And I see white spots in it which I tell you here.

If you see no problem here that's fine. I'm figuring what to do through trial-and-error anyway. And thanks for the project and your efforts regardless. :+1:
